clients/fides-js/src/lib/gpp/us-notices.ts

```diff
@@ -52,7 +52,10 @@ const deriveGppFieldRegion = ({
   experienceRegion: string;
   usApproach: GPPUSApproach | undefined;
 }) => {
-  if (usApproach === GPPUSApproach.NATIONAL) {
+  if (
+    experienceRegion?.toLowerCase().startsWith("us") &&
+    usApproach === GPPUSApproach.NATIONAL
+  ) {
```

Comment:
non-us regions were slipping through if force gpp was true. making sure it's a U.S. region before forcing this setting.

Reply:
is there a better / more robust way to check for "US" regions? If not can we at least add a util for this so it's more documented?